.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,41 @@
-## Proposed Changes
+Describe proposed changes and the issue being fixed
 
-  -
-  -
-  -
+<!-- Optional -->
+Fixes [link to the issue]
 
 ## Testing
+<!-- Optional -->
+Describe how you tested your changes. If possible and needed:
+- Test it on a sample project
+- Write unit tests
+- Provide a code snippet
 
-Test: Describe how you tested your changes. Note that this line (with `Test:`) is required, your PR will not build without it!
+<!-- Optional -->
+This should be tested by QA
 
-## Issues Fixed
+## Release Notes
+<!--
```

Comment:
A note: When viewing this file here (https://github.com/JetBrains/compose-multiplatform-core/blob/ac05bf267413ade149ef701b1fccdd1535f69c17/.github/PULL_REQUEST_TEMPLATE.md) It doesn 't show everything in  the brackets`<!-- ... -->`


Maybe it will work when creating a PR anyway.

Reply:
Yes, when creating a PR, we see it